src/java.base/share/classes/java/text/DecimalFormat.java

```diff
@@ -2437,6 +2441,9 @@ private final boolean subparse(String text, ParsePosition parsePosition,
         return true;
     }
 
+    // To track the full parse index as well as the integer portion
+    record Position(int intPos, int fullPos) {}
+
```

Comment:
The record name may be more descriptive. Also I would expect more description here, such as what it is and those params are.

Reply:
Thanks for the review Naoto.

Renamed to `NumericPosition`, as well as added further descritpion. Hesitant to rename to anything longer/detail too much, for what is essentially a limited scope helper class.